src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -926,6 +976,17 @@ private static JsValue ToStringWithFormat(JsValue self, JsValue[] args)
                         num.ToString(cultureInfo);
                 }
 
+                if (args[0].IsString())
+                {
+                    if (DateTime.TryParseExact(args[0].AsString(), "o", CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind, out var date) == false)
```

Comment:
I think that instead of 'o' we should use 'DefaultFormat.DateTimeFormatsToRead' ?